src/modules/app/routesMap.js

```diff
@@ -0,0 +1,26 @@
+import { I18N_REDIRECT_ROUTE, i18nRedirectRoute } from './routes/i18nRedirect'
```

Comment:
We only want to export classes and functions basically. Can we create a simple immutable RouteConfig/RouteMap from this?

Reply:
what is the problem with the current?

Reply:
It's plain-old javascript. We have now cool classes. I think this would be the only file which exports an object (except for the new route definitions)